src/components/Tool/FixPhoneNumbers/ContactPhoneNumbers.tsx

```diff
@@ -0,0 +1,234 @@
+  index,
+  person,
+  phoneNumber,
+  handleChangePrimary,
+  handleDeleteNumberOpen,
+}) => {
+  const { classes } = useStyles();
+  const { t } = useTranslation();
+  const locale = useLocale();
+  const { id: personId } = person;
+
+  return (
+    <>
+      <Grid
+        data-testid="phoneNumbers"
+        item
+        xs={6}
+        sm={4}
+        className={classes.paddingB2}
+      >
+        <Box
+          display="flex"
+          justifyContent="space-between"
+          className={classes.paddingX}
+        >
+          <Box>
+            <Hidden smUp>
+              <Typography display="inline" variant="body2" fontWeight="bold">
+                {t('Source')}:
+              </Typography>
+            </Hidden>
+            <Typography display="inline" variant="body2">
+              {`${sourceToStr(t, phoneNumber?.source)} (${dateFormatShort(
+                DateTime.fromISO(phoneNumber?.updatedAt),
+                locale,
+              )})`}
+            </Typography>
+          </Box>
+        </Box>
+      </Grid>
+      <Grid item xs={6} sm={2} className={classes.paddingB2}>
+        <Box
+          display="flex"
+          justifyContent="center"
+          className={classes.paddingX}
+        >
+          <Typography display="flex" alignItems="center">
+            {phoneNumber?.primary ? (
+              <>
+                <Hidden smUp>
+                  <Typography
+                    display="inline"
+                    variant="body2"
+                    fontWeight="bold"
+                  >
+                    {t('Source')}:
+                  </Typography>
+                </Hidden>
+                <StarIcon
+                  data-testid={`starIcon-${personId}-${phoneNumber?.id}`}
+                  className={classes.hoverHighlight}
+                />
+              </>
+            ) : (
+              <>
+                <Hidden smUp>
+                  <Typography
+                    display="inline"
+                    variant="body2"
+                    fontWeight="bold"
+                  >
+                    {t('Source')}:
+                  </Typography>
+                </Hidden>
+                <Tooltip title={t('Set as Primary')} placement="left">
+                  <StarOutlineIcon
+                    data-testid={`starOutlineIcon-${personId}-${phoneNumber?.id}`}
+                    className={classes.hoverHighlight}
+                    onClick={() => handleChangePrimary(personId, index)}
+                  />
+                </Tooltip>
+              </>
+            )}
+          </Typography>
+        </Box>
+      </Grid>
+      <Grid item xs={12} sm={6} className={classes.paddingB2}>
+        <Box
+          display="flex"
+          justifyContent="flex-start"
+          className={clsx(classes.responsiveBorder, classes.paddingX)}
+        >
+          <FormControl fullWidth>
+            <Field name={`numbers.${index}.number`}>
+              {({ field }: any) => (
```

Comment:
I believe the correct type is `FieldProps`.

```suggestion
              {({ field }: FieldProps) => (
```